hols/grafana/README.md

```diff
@@ -223,3 +399,39 @@ curl -s  \
   -w "%{http_code}\n" \
   "http://localhost:8080/greet/World?c=[1-500]"
 ```
+
+### Helidon SE
+
+Run the same curl command above but change the port to `8082`.
+
+## Start additional MP and SE JVM's
+
+Start an additional MP application JVM on port `8081` by running the following, and run curl requests against it.
+
+```bash
+java -Xmx1g -Xmx1g -Dserver.port=8081 -jar target/quickstart-mp.jar
+```
+
+You should see the total MP JVM count updated to 2, and when you navigate to the MP Details dashboards you 
+should see the additional JVM.
+
+Start and addition SE application on JVM port `8083`, by running the following, and explore the dashboards.
+
+```bash
+java -Xmx256m -Xmx256m -Dserver.port=8083 -jar target/quickstart-se.jar
+```
+
+> Note: The additional JVM's are picked up by prometheus as the [prometheus.yaml](prometheus.yaml) configuration
+> already contains the ports to scrape from.
+
+## Explore the Loki logs
+
+If you wish to explore the Loki logs you can do the following in Grafana:
+
+1. Open the Grafana menu at the top left, and click on `Explore`
```

Comment:
In my version of Prometheus (3.3.0) you can do this by selecting

Drilldown->Logs

Which appears to select the Loki datasource and run the query.